src/components/Header.js

```diff
@@ -21,16 +21,20 @@ class Header extends React.Component {
   constructor(props) {
     super(props);
     const { description, image, twitterHandle, title } = props;
+    const defaultTitle = title ? `${title} - Open Collective` : 'Open Collective';
+    const defaultImage = image ? image : 'https://opencollective.com/static/images/opencollectiveicon-240x240@2x.png';
```

Comment:
Huh. That's not the right variable name if you use it this way. 
If you name it `defaultImage`, then it should be a constant that is independent of any condition.

```js
const defaultImage = 'https://opencollective.com/static/images/opencollectiveicon-240x240@2x.png';
meta['twitter:image'] = image ? image : defaultImage;
meta['twitter:image'] = image || defaultImage; // better
```

Reply:
Understood

Reply:
<img width="756" alt="Screenshot 2019-07-12 at 16 58 59" src="https://user-images.githubusercontent.com/29008971/61141862-6c5c9f00-a4c6-11e9-9999-fe108e67a7bb.png">

is this alright?

Reply:
Much much better! :-)

Reply:
Alright, would push.